src/client/markdown.ts

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import type * as MarkdownIt from 'markdown-it';
+import type { RendererContext } from 'vscode-notebook-renderer';
+
+interface MarkdownItRenderer {
+    extendMarkdownIt(fn: (md: MarkdownIt) => void): void;
+}
+const css = `
+<style>
```

Comment:
Is it possible to avoid needing to duplicate this stylesheet?

Reply:
Can, but that just too complicated for something so simple

Reply:
Can you at least add a comment to the .css file that mentions "this should be kept in sync with this_js_file" and vice versa?

Reply:
It doesn't look like you added that comment